esp/services/ws_topology/ws_topologyService.cpp

```diff
@@ -1399,11 +1399,11 @@ bool CWsTopologyEx::onTpClusterInfo(IEspContext &context, IEspTpClusterInfoReque
         {
 #ifndef _CONTAINERIZED
             Owned<IConstWUClusterInfo> clusterInfo = getTargetClusterInfo(req.getName());
+            if (clusterInfo == nullptr)
+                throw MakeStringException(ECLWATCH_INVALID_CLUSTER_NAME, "Invalid Target Cluster name provided: '%s'", req.getName());
```

Comment:
shouldn't the #else block also check for null and throw same error?

Reply:
The #else block do not need the null check because it has been checked in the getContainerWUClusterInfoByName().